saleor/checkout/migrations/0054_alter_checkout_options.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.2.15 on 2022-08-26 08:14
+
+from django.apps import apps as registry
+from django.db import migrations
+from django.db.models.signals import post_migrate
+
+
+def assign_permissions(apps, schema_editor):
+    def on_migrations_complete(sender=None, **kwargs):
+        Permission = apps.get_model("auth", "Permission")
+        App = apps.get_model("app", "App")
+        Group = apps.get_model("auth", "Group")
+
+        handle_taxes = Permission.objects.filter(
```

Comment:
We should use the actual permission name. `handle_taxes` is separate permission. 
```suggestion
        manage_taxes = Permission.objects.filter(
```